aidial_adapter_vertexai/chat/static_tools.py

```diff
@@ -0,0 +1,82 @@
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import List, Self
+
+from aidial_sdk.chat_completion.request import (
+    AzureChatCompletionRequest,
+    StaticTool,
+)
+from pydantic import BaseModel
+from vertexai.preview.generative_models import Tool as GeminiTool
+from vertexai.preview.generative_models import grounding
+
+from aidial_adapter_vertexai.chat.errors import ValidationError
+
+
+class ToolName(str, Enum):
+    """
+    https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/grounding
```

Comment:
It's a great comment, but a poor docstring for this enum.

Imagine what it looks like in a generated documentation, not the source code.

Reply:
Fixed